app/js/displayEditModal.js

```diff
@@ -0,0 +1,28 @@
+let editModal = document.getElementById('modal')
+
+let modalButton = document.getElementById('modalBtn')
+
+let closeTag = document.getElementsByClassName('close')[0]
+
+let container = document.getElementsByClassName('container')[0]
+
+
+modalButton.onclick = function (){
```

Comment:
line 10 should have psr2 ,if you're gonna do it for other lines like line 23, it's best to do it throughout the code

Reply:
resolved